python/ray/autoscaler/autoscaler.py

```diff
@@ -503,14 +503,16 @@ def check_required(config, schema):
 def check_extraneous(config, schema):
     """Make sure all items of config are in schema"""
     if type(config) is not dict:
-        raise ValueError("Config is not a dictionary")
+        raise ValueError("Config {} is not a dictionary".format(config))
     for k in config:
         if k not in schema:
             raise ValueError(
                 "Unexpected config key `{}` not in {}".format(
                     k, list(schema.keys())))
         v, kreq = schema[k]
-        if isinstance(v, type):
+        if v is None:
```

Comment:
Maybe we should consider writing a test for validating the remote config generated...